### PR TITLE
2023Q3 office@ report

### PR DESCRIPTION
**Pull request by fluffykhv**

### CODE REVIEW

File: website/content/en/status/report-2023-07-2023-09/office.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Contact: FreeBSD Office team ML <office@FreeBSD.org> +
+Contact: Dima Panov <fluffy@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+The FreeBSD Office team works on a number of office-related software suites and tools such as OpenOffice and LibreOffice.
+
+Work during this quarter was focused on providing the latest stable release of LibreOffice suite and companion apps to all FreeBSD users.
+
+During the 2023Q3 period we pushed maintenance patches for the LibreOffice port and brought the latest, 7.6.2, release and all companion libraries such as MDDS, libIxion and more to the ports tree.
+All prerelease development of libreoffice ports is carried out in the in link:https://github.com/freebsd/freebsd-ports-libreoffice[LibreOffice WIP repo].
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/office.adoc:16]:
```suggestion
All prerelease development of LibreOffice ports is carried out in the in link:https://github.com/freebsd/freebsd-ports-libreoffice[LibreOffice WIP repo].
```

File: website/content/en/status/report-2023-07-2023-09/office.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Together with Libreoffice, we also updated Boost to the latest, 1.83 release.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/office.adoc:18]:
```suggestion
Together with LibreOffice, we also updated Boost to the latest, 1.83 release.
```

File: website/content/en/status/report-2023-07-2023-09/office.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Everyone interesteted in Boost porting can submit patches to the link:https://github.com/fluffykhv/freebsd-ports-boost[Boost WIP repository].
+
+We are looking for people to help with the open tasks:
+
+* The link:https://bugs.freebsd.org/bugzilla/buglist.cgi?bug_status=open&email1=office%40FreeBSD.org&emailassigned_to1=1&emailcc1=1&emailreporter1=1&emailtype1=substring&query_format=advanced&list_id=374316[open bugs list] contains all filed issues which need some attention
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/office.adoc:23]:
```suggestion
* The link:https://bugs.freebsd.org/bugzilla/buglist.cgi?bug_status=__open__&email1=office%40FreeBSD.org&emailassigned_to1=1&emailcc1=1&emailtype1=substring&list_id=650685&order=Bug+Number&query_format=advanced[open bugs list] contains all filed issues which need some attention
```
The resulting list of more than six hundred reports included many that are **not** open. 

Instead: a list of forty-three (at the time of writing).

**Review by grahamperrin — CHANGES_REQUESTED** [review]

**Comment by lsalvadore** [comment]
Merged, with @grahamperrin's comments.

**Pull request closed without merging**